### ISSUE TITLE
Typo "oldest" instead of "newest"

### ISSUE THREAD

**Issue by larsen0815** [FrankysWeb/Delete-Old-OWA-and-ECP-Versions#2]
write-host "Oldest installed Exchange Build is: $ExchangeServerVersion"
Shouldn't this be the newest installed Exchange version?

**Comment by Deckard99** [comment]
No, the oldest has to stay because of backward compatibility with "oldest" Exchange in your environment. If you only have one Exchange oldest or latest ist the same. ;)
